receiver/chronyreceiver/metadata.yaml

```diff
@@ -0,0 +1,68 @@
+---
+name: chrony receiver
+
+attributes:
+  leap.status:
+    description: how the chrony is handling leap seconds
+    type: string
+    enum:
+    - normal
+    - insert_second
+    - delete_second
+    - unsynchronised
+
+metrics:
+  ntp.frequency.offset:
+    enabled: false
+    description: The frequency is the rate by which the system s clock would be wrong if chronyd was not correcting it.
+    extended_documentation: It is expressed in ppm (parts per million). For example, a value of 1 ppm would mean that when the system\’s clock thinks it has advanced 1 second, it has actually advanced by 1.000001 seconds relative to true time.
+    unit: "{ppm}"
+    gauge:
+      value_type: double
+    attributes:
+    - leap.status
+  ntp.skew:
+    enabled: true
+    description: This is the estimated error bound on the frequency.
+    unit: "{ppm}"
```

Comment:
Parts per million is an actual UCUM defined unit (https://ucum.org/ucum.html). 

So it means you don't need the `{}`